src/content/docs/reference/vscode.mdx

```diff
@@ -1,140 +1,310 @@
 
-- Open the *extensions* tab (_View_ → _Extensions)_ and search for Biome.
-- Open the _Quick Open Overlay_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd> or _Go -> Go to File_), enter `ext install biomejs.biome`, and hit enter.
+## Installing the extension
 
-## Getting Started
+The extension is available both on the 
+[Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) and the 
+[Open VSX registry](https://open-vsx.org/extension/biomejs/biome) and can be installed directly
+by clicking one of the following links:
 
-### Default Formatter
+[![](https://img.shields.io/badge/Install%20Biome-in%20VS%20Code-007ACC?style=flat&logo=biome)](vscode:extension/biomejs.biome)
 
-Configure Biome as the default formatter for supported files to ensure that VS Code uses Biome over other formatters that you may have installed. You can do so by opening a JavaScript or TypeScript and then:
+[![](https://img.shields.io/badge/Install%20Biome-in%20VSCodium-007ACC?style=flat&logo=biome)](vscodium:extension/biomejs.biome)
 
-- Open the Command Palette (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> or View → Command Palette)
-- Select _Format Document With…_
-- Select _Configure Default Formatter…_
-- Select Biome
+## Projects
 
-You can also enable Biome for specific languages only:
+The Biome extension operates on so-called _projects_. A project is a directory in which the extension will spawn a dedicated
+Biome LSP server. This is useful because it allows you to use different versions of Biome for different projects, and to
+have different configurations for each project.
 
-- [Open the `settings.json`](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson): open the _Command Palette_(<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd>) and select _Preferences: Open User Settings (JSON)_
-- And set the `editor.defaultFormatter` to `biomejs.biome` for the desired language
+### Defining projects
 
-```json title="settings.json"
-{
-	"editor.defaultFormatter": "<other formatter>",
-	"[javascript]": {
-		"editor.defaultFormatter": "biomejs.biome"
-	}
-}
-```
+Projects can be defined in two places:
+
+- At the **workspace** level (`.code-workspace`)
+- At the **workspace folder** level (`.vscode/settings.json`)
+
+:::note[Implicit projects]
+The extension automatically creates a project at the root of every workspace folder that does not have explicitly defined projects.
+:::
+
+#### Defining projects in a workspace
+
+To define projects in a **workspace**, add a `settings.biome.projects` property to the `.code-workspace` file. This setting accepts an
+array of project definitions, each of which has the following properties:
+
+- `name`: The **name of the workspace folder** that the project is associated with
+- `path` : The **path to the project directory**, _relative to the root of the workspace folder_.
```

Comment:
At this point, it would be nice to show a `JSONC` example that people can visually see and maybe copy. The example will be helpful to clarify way more the `path`. Also, it isn't clear if both properties are optional or mandatory